src/util/rocksdb/properties.rs

```diff
@@ -466,6 +479,251 @@ impl DecodeProperties for UserCollectedProperties {
     }
 }
 
+#[derive(Debug, Default)]
+pub struct IndexHandlesMap {
+    pub handles: BTreeMap<Vec<u8>, Vec<IndexHandle>>,
+    size: usize,
+}
+
+pub struct IndexHandlesIterator<'a> {
+    iter: Iter<'a, Vec<u8>, Vec<IndexHandle>>,
+    idx: usize,
+}
+
+impl<'a> IndexHandlesIterator<'a> {
+    fn new(iter: Iter<'a, Vec<u8>, Vec<IndexHandle>>, idx: usize) -> IndexHandlesIterator<'a> {
+        IndexHandlesIterator { iter, idx }
+    }
+}
+
+impl<'a> Iterator for IndexHandlesIterator<'a> {
+    type Item = (&'a [u8], &'a IndexHandle);
+    fn next(&mut self) -> Option<(&'a [u8], &'a IndexHandle)> {
+        match self.iter.next() {
+            None => None,
+            Some((k, handles)) => Some((k, &handles[self.idx])),
+        }
+    }
+}
+
+impl IndexHandlesMap {
+    fn new(size: usize) -> IndexHandlesMap {
+        IndexHandlesMap {
+            handles: BTreeMap::new(),
+            size,
```

Comment:
Seems like this value will only be used for an assert in `get_approximate_distance_in_range`. Can it be removed?